tensorflow_lite_support/python/task/processor/proto/search_result_pb2.py

```diff
@@ -13,7 +13,90 @@
 # limitations under the License.
 """Search result protobuf."""
 
+import dataclasses
+from typing import Any, List
+
 from tensorflow_lite_support.cc.task.processor.proto import search_result_pb2
+from tensorflow_lite_support.python.task.core.optional_dependencies import doc_controls
+
+_NearestNeighborProto = search_result_pb2.NearestNeighbor
+_SearchResultProto = search_result_pb2.SearchResult
+
+
+@dataclasses.dataclass
+class NearestNeighbor:
+  """A single nearest neighbor.
+
+  Attributes:
+    metadata: User-defined metadata about the result. This could be a label, a
+      unique ID, a serialized proto of some sort, etc.
+    distance: The distance score indicating how confident the result is.
+      Lower is better.
+  """
+
+  metadata: bytearray
```

Comment:
Please convert this to string.

Reply:
@khanhlvg Fixed it.

Reply:
After revising the code, It turned out that bytearray is the correct type because users may want to use non-string metadata to save space. Therefore, I reverted the commit.

Reply:
@khanhlvg Yes, that was my observation too from the proto documentation so I kept it as a bytearray.